index.html

```diff
@@ -124,20 +124,20 @@ <h2 class="project-title">Tonic</h2>
                         <li>css</li>
                         <li>javasSript</li>
                     </ul>
-                    <button class="button" type="button">See Project</button>
+                    <button class="button work-button" type="button">See Project</button>
                 </div>
         </section>
-        <section class="work-card">
+        <section class="work-card reverse-card">
                 <img class="profile-image" src="images/art-printing.png" alt="professional art printing" width="295"
                     height="220">
                 <div class="left-block"> 
-                    <h2 class="project-title">Mult-Post Stories</h2>
+                    <h2 class="project-title">Uber Navigation</h2>
                     <ul class="credentials-list">
-                        <li>CANOPY</li>
+                        <li>Uber</li>
                         <li><i class="fa fa-circle" aria-hidden="true"></i></li>
-                        <li>Back End Dev</li>
+                        <li>Lead Developer</li>
                         <li><i class="fa fa-circle" aria-hidden="true"></i></li>
-                        <li>2015</li>
+                        <li>2008</li>
                     </ul>
```

Comment:
- [ ] Please let's make sure we use the correct `color`, `margin`, `alignment`, and `font-size` for the project cards. see the difference between your page and Figma's template.

Yours:
<img width="493" alt="Loom_7mFUm50qbl" src="https://user-images.githubusercontent.com/81468675/149596903-f12c7e65-4c35-479f-98c5-55d5d783db52.png">

Figma's:
<img width="374" alt="Loom_fo5y42wx7F" src="https://user-images.githubusercontent.com/81468675/149596924-f4a992c4-cd9d-4e94-b0ff-b86b8bc31cef.png">

Reply:
Resolved